### DISCUSSION TITLE
IE11 support not working?

### DISCUSSION THREAD

**Discussion by mschoenb** [mobxjs/mobx#2597] [Q&A] [answered]
Hi,

probably I'm doing somtehing wrong but I can't get mobx6 working with IE11. I've a React application and I'm using createReactApp and Typescript. In my index.tsx I've added

import "react-app-polyfill/ie11";
import "react-app-polyfill/stable";
import { configure } from "mobx";
configure({ useProxies: "never", enforceActions: "never" });

but IE11 doesn't work and the console log shows 

[MobX] `Proxy` objects are not available in the current environment. Please configure MobX to enable a fallback implementation.

Changing {enforceActions: "never"} to "always" will toggle strict-mode when watching the console log in Firefox. So the line is reached and configure will do somehting but it doesn't disable the proxies in IE11.

Any idea?

Regards
  Marcus

**Comment by urugator** [comment, marked as answer]
Can you please verify `mobx._getGlobalState().useProxies === false`?

Do you know when it throws? Does it throw if you try to create observable object or array. Or does it throw immediately on import?

**Reply by mschoenb** [reply]
I don't really understand what's going on here. I tried it several times during the last 2 days and it didn't work. Yesterday evening it works after removing the ; at the end of configure. I've added and removed it this morning and it's still working in both situations. So currently eveything is ok.

**Comment by mweststrate** [comment]
The problem and solution might be caused by making an unrelated code change. JavaScript module loading order is not very transparent, so it might be very well the case that another module is (indirectly) imported and run before arriving at the configure call. A simple way to check that is to put a `debugger` statement or throwing an exception just before the `configure` call. If you don't arrive there, the issue is that some imported code is already using MobX before it is configured
